apiserver/plane/bgtasks/webhook_task.py

```
@@ -402,8 +402,10 @@ def webhook_activity(
 
         if event == "issue_comment":
             webhooks = webhooks.filter(issue_comment=True)
-
+            
         for webhook in webhooks:
+            event_data = get_model_data(event=event, event_id=event_id)
```

Ruff (0.8.2)

Local variable `event_data` is assigned to but never used

Remove assignment to unused variable `event_data`

(F841)